src/auto/accel_group.rs

```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (b7f5189) from gir-files (71d73f0)
+// This file was generated by gir (b7f5189) from gir-files (71d73f0+)
```

Comment:
Remove gir.exe from target. Maybe it detect that no changes and not rebuild `gir_version.rs`